examples/extensions/update_sitelink.py

```diff
@@ -79,7 +79,7 @@ def main(client, customer_id, feed_item_id, sitelink_text):
 
     parser = argparse.ArgumentParser(
         description="Update sitelink extension feed item with the specified "
-        "link text and URL."
```

Comment:
Copied from the golden originally. The example does not actually reference URL (I flagged this to the golden owner as well)

Reply:
I suggest you fix it here while waiting for the golden to be fixed.